python/ray/log_monitor.py

```diff
@@ -33,6 +33,7 @@ def __init__(self,
         self.file_position = file_position
         self.file_handle = file_handle
         self.worker_pid = None
+        self.driver_id = None
```

Comment:
Let's call this `current_driver_id`. Can you also add a comment on the previous line explaining what the purpose of this field is?